### ISSUE TITLE
Screenreader accessibility has seriously regressed in v0.16

### ISSUE THREAD

**Issue by robin24** [stalwartlabs/webui#9]
I'm a fully blind admin and developer, using a screenreader to interface with my computer using text to speech as well as Braille output.

Unfortunately, after upgrading my Stalwart deployment to v0.16, I've had to discover that the new, redesigned web UI has serious accessibility regressions, making it nearly impossible to access with a screenreader. This strongly contrasts the old web UI in previous versions, which was pretty much perfectly accessible. I've now downgraded back to v0.15 as a result.

Here are some of the issues I ran into after upgrading:
* Several buttons across the UI (e.g. Add, Delete, etc) are missing accessibility labels, resulting in screenreaders simply announcing them as "Button", making it completely unclear what the button actually does.
* The states of checkboxes are not exposed to screenreaders, making it completely impossible to tell whether a feature / setting / etc. is currently toggled on or of. I was able to tell that a checkbox was there, but they were merely being announced as "checkbox" without on / off state, pressing Space on one of them, which would normally result in the checkbox being toggled and the screenreader announcing the state change as a result appeared to have no effect in the Stalwart UI.
* The contents of tables are completely hidden from screenreaders. As a result, it is now impossible to browse lists of e.g. domains, users, etc. This issue manifests itself in the form of a screenreader announcing that there is a table (including row and column count) when the table is focused, but then not being able to navigate inside that table to review what it contains.
* Modal dialogs don't receive focus automatically. When a dialog shows up on the page, e.g. when leaving a settings page with unsaved changes, focus doesn't move to it automatically. As a result, it is difficult to become aware of the fact that a dialog has just appeared, unless explicitly navigating around the page to search for any changes.

Tested with VoiceOver on macOS 26, using both Safari and Chrome.

I really hope these issues can be addressed before the first stable release of Stalwart, as unfortunately the web UI is pretty much unusable for customers such as myself. This should also be a priority for enterprise customers and potential government contractors wanting to migrate to Stalwart, as they will need to comply with accessibility regulations.

Happy to answer any questions as best I can.
